dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/models/SubSnpCoreFields.java

```diff
@@ -379,12 +386,20 @@ public Region getVariantCoordinates() {
         return variantRegion;
     }
 
+    private boolean isValidRegion(Region region) {
+        return region != null && region.getChromosome() != null && region.getStart() != null && region.getEnd() != null;
+    }
+
     /**
      * Return the left aligned, normalised, variant coordinates and the alleles in the forward strand
      * @return Object containing normalised variant coordinates and forward strand alleles
      */
     public VariantCoreFields getVariantCoreFields() {
         Region variantRegion = getVariantCoordinates();
+        if (variantRegion == null) {
+            throw new IllegalArgumentException(
+                    "DbSNP variants without coordinates are not expected to be inserted as EVA Variants");
```

Comment:
The insertion or other task is not something this class should be aware of, so I would suggest changing this message to:

```
dbSNP variants without coordinates are not valid EVA variants
```